packages/extension-chakra-store-locator/src/components/with-optional-commerce-sdk-react-provider.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import {getAppOrigin} from '@salesforce/pwa-kit-react-sdk/utils/url'
+import {CommerceApiProvider} from '@salesforce/commerce-sdk-react'
+import {UserConfig} from '../types/config'
+
+type WithOptionalCommerceSdkReactProvider = React.ComponentPropsWithoutRef<any>
```

Comment:
I know this comment isn't related to what you are implementing in your PR.. but seeing `any` triggered me to ask a question.

Reply:
I'm not sure if there is a better option for this, maybe use `unknown`? since the underlying wrapped component props is unknown?